img/tools/link.svg

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
Can you pass this SVG into something like [SVGOMG](https://jakearchibald.github.io/svgomg/) to remove all the unnecessary stuff such as these empty `<g>` elements?

Reply:
Great tool; thanks for informing me about it!